releasenotes/notes/apm-add-orchestrator-tag-f16b9cc1d55ac082.yaml

```
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added Orchestrator tag to profile data if host is on AWS Fargate.
+    Important for properly tracking profile data in Fargate, which might run
+    on a large pool of servers. Helpful for billing.
```

This is too much detail for the user. Also, we need to prefix trace-agent related release notes with `APM: `. Maybe try something like this:
```suggestion
    APM: Added additional tags to profiles in AWS Fargate environments.
```
This should be plenty. At most, if you wish, you can mention which tags (but I assume they are not really relevant to the user).